lib/widgets/screens/dashboard/dashboard.dart

```diff
@@ -50,6 +51,10 @@ class DashboardScreen extends StatelessWidget {
                   profileMessage: _getGreeting(l10n, user),
                   profileCompletionPercentage: 1.00,
                 ),
+                MaybeReminderBanner(
+                  profileCompletionPercentage: 30,
```

Comment:
Note that the profileCompletionPercentage for the profile progress indicator is a double from 0.0 to 1.00, whereas the reminder banner is using an int from 0 to 100. Either approach is fine but we shouldn't mix both. You can go ahead and change the logic in the progress indicator if you think an int makes more sense. I just chose that because that is what the default widget uses.

Reply:
Changed the profile icon to using integer